kahuna/public/js/directives/ui-crop-box/ui-crop-box.js

```diff
@@ -83,11 +87,40 @@ cropBox.directive('uiCropBox', ['$timeout', '$parse', 'safeApply', 'nextTick', '
                 });
             }
 
+            function getPlayButton() {
+                const playButtonTpl = `
+                    <div class="video-play-button video-play-button--hidden">
+                        <gr-icon>play_arrow</gr-icon>
+                    </div>`;
+
+                const templateEl = document.createElement('template');
+                templateEl.innerHTML = playButtonTpl;
+
+                return templateEl.content.firstElementChild;
+            }
+
+            function appendPlayVideoEl() {
+                const cropBoxEl = cropper.cropBox;
+                scope.playButtonEl = getPlayButton();
+                cropBoxEl.appendChild(scope.playButtonEl);
```

Comment:
`appendChild` rather than transcluding the element in the traditional angular way to make it move as the cropBox moves - that way we don't have to have listeners on the position of the cropBox and then move the play button accordingly.